command/ca/admin/list.go

```diff
@@ -0,0 +1,122 @@
+package admin
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/smallstep/certificates/linkedca"
+	"github.com/smallstep/cli/errs"
+	"github.com/smallstep/cli/flags"
+	"github.com/smallstep/cli/utils/cautils"
+	"github.com/urfave/cli"
+)
+
+func listCommand() cli.Command {
+	return cli.Command{
+		Name:   "list",
+		Action: cli.ActionFunc(listAction),
+		Usage:  "list all admins in the CA configuration",
+		UsageText: `**step ca admin list** [**--super**] ]**--not-super**] [**--provisioner**=<string>]
+[**--ca-url**=<uri>] [**--root**=<file>]`,
+		Flags: []cli.Flag{
+			flags.CaURL,
+			flags.Root,
+			cli.BoolFlag{
+				Name:  "super",
+				Usage: `Only return super-admins.`,
+			},
+			flags.X5cCert,
+			flags.X5cKey,
+			flags.PasswordFile,
+			cli.BoolFlag{
+				Name:  "not-super",
+				Usage: `Only return admins without 'super' privileges.`,
+			},
+			cli.StringFlag{
+				Name:  "provisioner",
+				Usage: `Only return admins linked to this provisioner.`,
+			},
+		},
+		Description: `**step ca admin list** lists all admins in the CA configuration.
+
+## EXAMPLES
+
+List all admins:
+'''
+$ step ca admin list
+'''
+
+List only super-admins:
+'''
+$ step ca admin list --super
+'''
+
+List only admins without super-admin privileges:
+'''
+$ step ca admin list --not-super
+'''
+
+List all admins associated with a given provisioner:
+'''
+$ step ca admin list --provisioner admin-jwk
+'''
+
+List only super-admins associated with a given provisioner:
+'''
+$ step ca admin list --super --provisioner admin-jwk
+'''
+`,
+	}
+}
+
+func listAction(ctx *cli.Context) (err error) {
+	if err := errs.NumberOfArguments(ctx, 0); err != nil {
+		return err
+	}
+
+	isSuperAdmin := ctx.IsSet("super")
+	isNotSuperAdmin := ctx.IsSet("not-super")
```

Comment:
ctx.Bool("xxx")

Reply:
Optionally you can also remove not-super, because it can be done with:
```sh
step-dev beta ca admin list --x5c-cert x5c.crt --x5c-key x5c.key --super=false
```

And probably `isNotSuperAdmin := ctx.IsSet("super") && ctx.Bool("super")` or something similar combining IsSet and Bool